pretty.js

```diff
@@ -46,6 +46,10 @@ function filter (value) {
   return result
 }
 
+function isPinoLine (line) {
+  return line.hasOwnProperty('hostname') && line.hasOwnProperty('pid')
+}
```

Comment:
If `v` is even present it's currently at 1.

What I am basing the conditions on is https://github.com/mcollina/pino/blob/master/pretty.js#L100 . Without those properties `pretty.js` will not construct a valid reformatted log line. The `v` property is not considered at all for reformatting.